lib/init/grass.py

```diff
@@ -1382,7 +1382,7 @@ def set_language(grass_config_dir):
     else:
         for line in fd:
             if re.search("^language", line):
-                line = line.rstrip(" %s" % os.linesep)
+                line = line.rstrip(" %s\n")
```

Comment:
@wenzeslaus My bad. Removed ``%s``, but not sure what you mean by *this change*. This PR or the above line 1385?

Reply:
See the conflict. This can't be merged because this change conflicts with main. However, just remove your change, commit that, and the conflict should disappear. The problem will disappear too because the line is simply not there anymore.